holidays/countries/barbados.py

```diff
@@ -0,0 +1,70 @@
+from holidays.calendars.gregorian import JAN, APR, JUL, AUG, NOV
+from holidays.holiday_base import HolidayBase
+from holidays.holiday_groups import ChristianHolidays, InternationalHolidays
+
+
+class Barbados(HolidayBase, ChristianHolidays, InternationalHolidays):
+    """
+    https://en.wikipedia.org/wiki/Public_holidays_in_Barbados
+    https://www.timeanddate.com/holidays/barbados/
+    """
+
+    country = "BB"
+
+    special_holidays = {
+        # One off 50th Anniversary of CARICOM Holiday.
+        # See https://tinyurl.com/brbhol
+        2023: (JUL, 31, "50th Anniversary of CARICOM Holiday")
+    }
+
+    def __init__(self, *args, **kwargs) -> None:
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        super().__init__(*args, **kwargs)
+
+    def _populate(self, year):
+        super()._populate(year)
+
+        # New Year's Day
+        self._add_new_years_day("New Year's Day")
+
+        # Errol Barrow Day
+        self._add_holiday("Errol Barrow Day", JAN, 21)
```

Comment:
> self._add_holiday_1st_mon_of_aug("Kadooment Day")

The same way you can now use 

```suggestion
        self._add_holiday_jan_21("Errol Barrow Day")
```

type of methods for month/day holidays and get rid of the 

`from holidays.calendars.gregorian import JAN, APR, AUG, NOV` in the import line above.

Reply:
Hey @arkid15r, thank you! It went pretty okay, but you know, the actual work begins after the move, but we're almost there! :) 

Just made the suggested changes and pushing it now after testing.

Reply:
And there we go @arkid15r, tests passed :)